question03.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <title>TerraceTechフロントエンドエンジニア養成所</title>
+  </head>
+  <body>
+    <div>
+      <ul></ul>
+    </div>
+  </body>
+  <script>
+      const ul = document.querySelector('ul');
+      for (let i = 0; i < 2; i++){
```

Comment:
この2は何の2だか分からないので
説明変数にしたいです
```suggestion
      const length = 2; // LengthOfElementsToBeCreated でもいいですけどここではそこまでは求めないことにします
      for (let i = 0; i < length; i++){
```